gradle.properties

```diff
@@ -5,23 +5,23 @@ pluginName = kphpstorm
 pluginRepositoryUrl = https://github.com/VKCOM/kphpstorm
 
 # SemVer format -> https://semver.org
-pluginVersion = 1.2.11
+pluginVersion = 1.2.12
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 233
-pluginUntilBuild = 233.*
+pluginSinceBuild = 241
+pluginUntilBuild = 241.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = IU
-platformVersion = 2023.3
+platformVersion = 2024.1
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 # PHP Plugin - https://plugins.jetbrains.com/plugin/6610-php/versions
-platformPlugins = com.jetbrains.php:233.11799.232
+platformPlugins = com.jetbrains.php:241.14494.237
 
 # Gradle Releases -> https://github.com/gradle/gradle/releases
-gradleVersion = 8.5
+gradleVersion = 8.6
```

Comment:
maybe 8.7?

Reply:
No. Let's not update earlier than we did in [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)

Reply:
No. Let's not update earlier than we did in [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)